### ISSUE TITLE
ED-209 Security Robots don't patrol on Tram Station

### ISSUE THREAD

**Issue by General-Gaston** [tgstation/tgstation#87958]
### Client Version:

515.1646

### Issue Summary:

ED-209 Security Robots patrol feature automatically turns off when set to patrol the station even when the AI turns them on

### Round ID:

242320

### Test Merge Information:

- [[tgui] Tgui core components D-M](https://www.github.com/tgstation/tgstation/pull/87763)
- [Guncode Agony 4: The Great Projectile Purge](https://www.github.com/tgstation/tgstation/pull/87740)
- [[TM ONLY] Buffs jetpack/wing forces and stabilization greatly to see how major the subsystem chugging issue is](https://www.github.com/tgstation/tgstation/pull/87742)
- [Batons have their stamina damage reduced by armor, get stun armor penetration](https://www.github.com/tgstation/tgstation/pull/87754)
- [Add ignition sound and roaring fire to atmos fire](https://www.github.com/tgstation/tgstation/pull/84280)

### Reproduction Steps:

Make an ED-209 Security Robot, attempt to put it on patrol. The robot will turn it off in the same second

**Comment by Profakos** [comment]
I think only the Tramhole has nav beacons.

**Comment by Kurgis** [comment]
Yep, there's no nav routes on Tram's top level, and if there were, they'd likely be more inclined to conga line in to the tram, or reach incompatible pathfinding issues. Perhaps some routes could be set-up, but they'd have to have some creative pathing to avoid all the bolted airlocks. If we utilize nav routes through Tram's maintenance it should at least become possible.
